README.md

```diff
@@ -62,33 +62,36 @@ Thank you to all those who have translated asbplayer:
 If you are a non-English native, and would like to help translate asbplayer, please contact me on [Discord](https://discord.gg/ad7VAQru7m).
 
 ## Getting Started
+ - *Note: asbplayer can either be used purely as a tool for subtitle control or as a deck mining tool (in which case you will need the subtitle functionality to mine).
+If you want to strictly learn how to use asbplayer's subtitle features and are not interested in sentence or vocabulary mining, you can read about them under the [Detailed Usage](#detailed-usage) header.*

```

Comment:
It's true that the information about the subtitle features can be found under "Detailed usage" but the reader still has to read and hunt for it since a lot of the documentation is still about making flashcards. So I would suggest these changes:
- Be more specific, for example: "just follow step 6."
- Make the "subtitle features" below "Detailed usage" more obvious such as by grouping them together or adding more information. Currently the only features explicitly documented are subtitle detection, offset, and regex filtering.